src/sparsezoo/v2/test.md

```diff
@@ -0,0 +1,100 @@
+# API User pathway
+
+## Download one file
+
+### Through the string arguments
```

Comment:
IMO the string query param arguments shouldn't restrict `model.download()` functionality to only grab those files. instead, should just filter the relevant ones. (ie if recipe=transfer_learn, then `model.download` downloads everything but for checkpoints and recipes only includes the transfer learn ones)

for downloading specific directories/files - we can filter via the query params that were passed in

Reply:
ok, sounds good.